src/darwin/Framework/CHIP/CHIPControllerAccessControl.mm

```diff
@@ -32,6 +32,15 @@
 // CHIPIMDispatch.mm.
 constexpr EndpointId kSupportedEndpoint = 0;
 
+class DeviceTypeResolver : public Access::AccessControl::DeviceTypeResolver {
+public:
+    bool IsDeviceTypeOnEndpoint(chip::DeviceTypeId deviceType, chip::EndpointId endpoint) override
+    {
+        // We only allow access to the OTA software update provider.
+        return false;
```

Comment:
This is fine, but could also just include `InteractionModelEngine.h` and call `app::IsDeviceTypeOnEndpoint(deviceType, endpoint)`, right?  Then if that function starts returning something other than `false` this will work automatically.

Reply:
OK done.